extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -79,9 +79,14 @@ populations and will also be returned by the function `Models()`.
     ipop1 = nest.Create("inh_iaf_neuron", 30)
     ipop2 = nest.Create("inh_iaf_neuron", 30)
 
-Once you have set up your populations of neurons, the function `PrintNetwork()`
```

Comment:
In this case, users might perceive the output of `PrintNetwork()` as confusing rather than helpful, since the two populations of 30 neurons each would show up as one range of 60 neurons. With subnet removal in NEST3, `PrintNetwork()` will become even less useful. So I think we might as well drop it here and rather add new suggestions when we revise documentation for NEST3.